src/DevMenu/ui/Augmentations.tsx

```diff
@@ -39,50 +38,46 @@ export function Augmentations(props: IProps): React.ReactElement {
     props.player.augmentations = [];
   }
 
+  function clearQueuedAugs(): void {
+    props.player.queuedAugmentations = [];
+  }
+
   return (
     <Accordion TransitionProps={{ unmountOnExit: true }}>
-      <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+      <AccordionSummary expandIcon={<ExpandMore />}>
         <Typography>Augmentations</Typography>
       </AccordionSummary>
       <AccordionDetails>
-        <table>
-          <tbody>
-            <tr>
-              <td>
-                <Typography>Aug:</Typography>
-              </td>
-              <td>
-                <Select
-                  onChange={setAugmentationDropdown}
-                  value={augmentation}
-                  startAdornment={
-                    <>
-                      <IconButton onClick={queueAllAugs} size="large">
-                        <ReplyAllIcon />
-                      </IconButton>
-                      <IconButton onClick={queueAug} size="large">
-                        <ReplyIcon />
-                      </IconButton>
-                    </>
-                  }
-                  endAdornment={
-                    <>
-                      <IconButton onClick={clearAugs} size="large">
-                        <ClearIcon />
-                      </IconButton>
-                    </>
-                  }
-                >
-                  {Object.values(AugmentationNames).map((aug) => (
-                    <MenuItem key={aug} value={aug}>
-                      {aug}
-                    </MenuItem>
-                  ))}
-                </Select>
-              </td>
-            </tr>
-          </tbody>
-        </table>
+        <Select
+          onChange={setAugmentationDropdown}
+          value={augmentation}
+          startAdornment={
+            <>
+              <IconButton onClick={queueAllAugs} size="large">
+                <ReplyAll />
+              </IconButton>
+              <IconButton onClick={queueAug} size="large">
+                <Reply />
+              </IconButton>
+            </>
+          }
+          endAdornment={
+            <>
+              <IconButton onClick={clearAugs} size="large">
+                <Clear />
+              </IconButton>
+            </>
+          }
+        >
+          {Object.values(AugmentationNames).map((aug) => (
+            <MenuItem key={aug} value={aug}>
+              {aug}
+            </MenuItem>
+          ))}
+        </Select>
+        <Button sx={{ display: "block" }} onClick={clearQueuedAugs}>
+          Clear Queued Augmentations
```

Comment:
should we add note explaining you dont get refunded?

Reply:
I think it's fine. They're already in the dev menu, it's not like they're losing anything :p

Reply:
oh derp good point 😆